### DISCUSSION TITLE
Browser extension feature (/plugin)

### DISCUSSION THREAD

**Discussion by Matix-Media** [2fast-team/2fast#168] [Ideas]
I think it would be really cool if this extension would support a browser plugin that can pull a 2fa code from within a browser by just clicking the 2fa code box. 

I don't know if this a is a feature that would make it into the app itself, but it could maybe be integrated through a plugin system in the app.

**Comment by Sunbeam-ops** [comment]
Yes, but does the screenshot protection work in the browser?
I'm better off having the codes in the app.
Copy and paste, it's also quick
